cmd/msrc/generate.go

```diff
@@ -59,8 +59,16 @@ func main() {
 		panicif(err)
 	}
 
+	knownVulnsToRemove := map[string]struct{}{
```

Comment:
probably want to filter these to the advisory that mentioned them. Otherwise, in the event that these get assigned to another Windows vuln, we don't wind up with a maddeningly subtle bug.

Reply:
@jkatz01 Probably need to add a flag for that. https://github.com/fleetdm/vulnerabilities/blob/d84fcc377cf054ccf590c94a1e32bbc28ad37a3b/.github/workflows/generate-cve.yml#L10-L14 does this for our other vulns feed, so adding something similar for nvd seems like a solid option.